src/main/steps/applicant1/hub-page/sole/soleTemplateChooser.ts

```diff
@@ -0,0 +1,54 @@
+import { State } from '../../../../app/case/definition';
+import { StateSequence } from '../../../state-sequence';
+
+export const getSoleHubTemplate = (
+  displayState: StateSequence,
+  isServiceApplicationGranted: boolean,
+  isSuccessfullyServedByBailiff: boolean,
+  isAlternativeService: boolean
+): string => {
```

Comment:
why not just pass the userCase to the method and work out the different booleans in this function, since they're only used in this function

Reply:
I checked and isServiceApplicationGranted and alternativeServiceType (which sets the isAlternativeService) are both used elsewhere, so it's better to set them in the sole content file